silc/md/collective_variables.py

```diff
@@ -96,6 +96,18 @@ def distance_sum3(r1, r2, r3):
     r3 = barycenter(r3)
     return distance(r1, r2) + distance(r1, r3) - distance(r2, r3)
 
+class MultipliedDistances(FourPointCV):
```

Comment:
Just being picky. But can we call it `DistancesProduct`?

Reply:
Sounds definitely better. Done!